src/app/crypto.service.ts

```diff
@@ -172,4 +173,38 @@ export class CryptoService {
 
     return bs58check.encode(prefixAndKey);
   }
+
+  // NOTE: Our ETH address uses the bitcion/bitclout derivation path, not the ETH path.
+  // This is ugly but we only do it because they're just deposit addresses and we couldn't
+  // backfill this data for existing users because we store the derived private key.
+  // A user can still easily import their account to an ETH client to recover any ETH
+  // by coping the seedHex from local storage.
+  //
+  // We aren't using ethereumjs-util to minimize bundle size and protect from vulnerabilities
+  // until LavaMoat is ready
+  //
+  // Reference implementation: https://github.com/ethereumjs/ethereumjs-util/blob/master/src/account.ts#L249
```

Comment:
Thanks for linking to this reference implementation.  Super helpful!

Reply:
+1